core-blocks/paragraph/index.js

```diff
@@ -314,6 +322,8 @@ export const settings = {
 
 	attributes: schema,
 
+	toolbar,
```

Comment:
If we introduce a `toolbar` API like this (maybe `controls` is better thought), Can we just automatically apply it to the block `edit` function if available?

Reply:
I really like this idea. I'll see what I can get working :+1: